### PR TITLE
Python developer page

### PR DESCRIPTION
**Pull request by technige**
Don't merge yet. Still some missing detail and logos.

### CODE REVIEW

File: language-guides/python/python.adoc [hunk 1]
```diff
@@ -3,111 +3,238 @@
...
 :toc-title: Overview
-:toclevels: 2
+:toclevels: 1
 :section: Develop with Neo4j
 :section-link: language-guides
 
 .Goals
 [abstract]
-If you are a Pythonista, this guide provides an overview of options for connecting to Neo4j. While this guide is not comprehensive it will introduce the different drivers and link to the relevant resources.
+This guide provides an overview of how to connecting to Neo4j from Python. While it is not comprehensive, it aims to
```
Comment (whatSocks) [inline language-guides/python/python.adoc]: For easier versioning, these adocs have the rule of "one line per sentence", so this line would be split in two.

File: language-guides/python/python.adoc [hunk 2]
```diff
@@ -3,111 +3,238 @@
...
 
+image::http://dev.assets.neo4j.com.s3.amazonaws.com/wp-content/uploads/2014/06/python-logo-master-v3-TM.png[width=300,float="right"]
+
 toc::[]
 
 // tag::intro[]
-=== Neo4j from Python
-
-image::http://dev.assets.neo4j.com.s3.amazonaws.com/wp-content/uploads/2014/06/python-logo-master-v3-TM.png[width=300]
+For anyone working with https://www.python.org/[Python], the Neo4j community have contributed a range of driver
```
Comment (whatSocks) [inline language-guides/python/python.adoc]: more lines to split up here

**Comment by jexp** [comment]
Hey Nigel, just wanted to check with you on this? Please don't forget to finish it off :)

**Comment by jexp** [comment]
merged manually, thanks a lot.

**Pull request closed without merging**